web-back/lore/[name].ts

```diff
@@ -40,6 +42,14 @@ app.put(
                 timestamp: new Date().toISOString()
             });
         }
+        if ((image !== "" && !isValidUrl(image)) || (thumb !== "" && !isValidUrl(thumb))) {
```

Comment:
Prefer adding the `image !== ""` condition inside the isValidUrl() function to simplify this line